apps/dotcms-block-editor/.storybook/preview.js

```diff
@@ -0,0 +1,3 @@
+import { defineCustomElements } from '@dotcms/dotcms-webcomponents/loader';
+
+defineCustomElements();
```

Comment:
👀

Reply:
It allows us to use the web components in the block editor.